torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -2560,8 +2579,29 @@ Tensor TensorExprKernel::computeValue(const torch::jit::Value* v) {
   } else {
     argInputs.push_back(toArg(inputs[0]));
   }
+
   auto outputType = findDtypeForValue(v->node()->output());
   std::vector<ExprHandle> outputShape = sizesForValue(v);
+  // handle ops optional arguments
+  switch (op) {
+    case aten::conv2d: {
+      // handle optional bias
+      if (c10::get_if<ArgNone>(&argInputs[2])) {
```

Comment:
Can we handle this without explicitly generating the bias tensor? Is this essential for the external call version?

The version of `conv2d` that NNC supports can handle cases without bias (https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/tensorexpr/operators/conv2d.h#L19), however, the caller is not doing the right thing which needs to be fixed.

Reply:
Thanks for adding a TODO.